web/manifest.webmanifest

```diff
@@ -0,0 +1,30 @@
+{
+  "short_name": "@NAME@",
```

Comment:
I don't think these variable names are going to get picked up because the dist script doesn't look at this location.